internal/pkg/deploy/cloudformation/project.go

```diff
@@ -439,3 +439,45 @@ func (cf CloudFormation) waitForStackSetOperation(stackSetName, operationID stri
 		time.Sleep(3 * time.Second)
 	}
 }
+
+// DeleteProject deletes all project specific StackSet and Stack resources.
+func (cf CloudFormation) DeleteProject(name string, accounts, regions []string) error {
+	if _, err := cf.client.DeleteStackInstances(&cloudformation.DeleteStackInstancesInput{
+		Accounts:     aws.StringSlice(accounts),
+		RetainStacks: aws.Bool(false),
+		Regions:      aws.StringSlice(regions),
```

Comment:
Let's say that we have `env_A` deployed in `account=1,region=us-west-2` and `env_B` in `account=2,region=us-east-1`.

Will this call try to delete a stack in `account=1,region=us-east-1`? In that situation does it fail?

Reply:
So here's the scenario I tested:
```
ecs-preview project init myproj
// profile1 has creds for account A and region us-west-2
ecs-preview env init --name env1 --profile profile1
// profile2 has creds for account B and region us-east-2
ecs-preview env init --name env2 --profile profile2
ecs-preview app init -n myapp -d ./Dockerfile -t "Load Balanced Web App"
ecs-preview app deploy --name myapp --env env1
ecs-preview app deploy --name myapp --env env2

ecs-preview project delete --yes
✔ Deleted app myapp from env env1.
✔ Deleted app myapp from env env2.
✔ Deleted app myapp resources from project myproj.
✔ Deleted app myapp from project myproj.
✔ Deleted project resources.
✔ Deleted environment env1 from project myproj.
✘ Failed to delete environment env2 from project myproj: failed to find a stack named myproj-env2.
✔ Deleted project parameters.
✔ Deleted local workspace folder.
```

The StackSet instance deletion works fine across multiple accounts and regions. Environment deletion for the first environment worked, but failed for the second environment. At a glance it seems related to deleting environments requiring specific profiles, but need to dig into the details of why that is. It seems strange that one succeeded.

Reply:
The stop-gap for now is to use interactivity to enable the user to select a profile for environment deletion:
```
ecs-preview project delete

? Are you sure you want to delete project myproj? Yes
✔ Deleted app myapp from env env1.
✔ Deleted app myapp from env env2.
✔ Deleted app myapp resources from project myproj.
✔ Deleted app myapp from project myproj.
✔ Deleted project resources.

? Which named profile should we use to delete env1? one
✔ Deleted environment env1 from project myproj.

? Which named profile should we use to delete env2? two
✔ Deleted environment env2 from project myproj.
✔ Deleted project parameters.
✔ Deleted local workspace folder.
```